main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -0,0 +1,1096 @@
+        setCoordValue(gp.getLonDeg(), bLonDeg_100, bLonDeg_010, bLonDeg_001);
+        setCoordValue(gp.getLonMin(), bLonMin_10, bLonMin_01);
+        setCoordValue(gp.getLonSec(), bLonSec_10, bLonSec_01);
+
+        latButtons = Arrays.asList(bLatDeg_100, bLatDeg_010, bLatDeg_001,
+                                                 bLatMin_10,  bLatMin_01,
+                                                 bLatSec_10,  bLatSec_01,
+        bLatPnt_10000, bLatPnt_01000, bLatPnt_00100, bLatPnt_00010, bLatPnt_00001);
+
+        lonButtons = Arrays.asList(bLonDeg_100, bLonDeg_010, bLonDeg_001,
+                                                 bLonMin_10,  bLonMin_01,
+                                                 bLonSec_10,  bLonSec_01,
+        bLonPnt_10000, bLonPnt_01000, bLonPnt_00100, bLonPnt_00010, bLonPnt_00001);
+
+        coordButtons = new ArrayList<>(latButtons.size() + lonButtons.size());
+        coordButtons.addAll(latButtons);
+        coordButtons.addAll(lonButtons);
+
+        minButtons = Arrays.asList(tLatDegChar, bLatMin_10, bLatMin_01,
+                                   tLonDegChar, bLonMin_10, bLonMin_01);
+        secButtons = Arrays.asList(tLatMinChar, bLatSec_10, bLatSec_01,
+                                   tLonMinChar, bLonSec_10, bLonSec_01);
+        pointLowButtons = Arrays.asList(bLatPnt_00010, bLatPnt_00001,
+                                        bLonPnt_00010, bLonPnt_00001);
+        lastUnits = Arrays.asList(tLatLastUnits, tLonLastUnits);
+
+        bLatDeg_100.setNextButton(bLatDeg_010);
+        bLatDeg_010.setNextButton(bLatDeg_001);
+        bLatDeg_001.setNextButton(bLatMin_10);
+        bLatMin_10.setNextButton(bLatMin_01);
+        bLatMin_01.setNextButton(bLatSec_10);
+        bLatSec_10.setNextButton(bLatSec_01);
+        bLatSec_01.setNextButton(bLatPnt_10000);
+        bLatPnt_10000.setNextButton(bLatPnt_01000);
+        bLatPnt_01000.setNextButton(bLatPnt_00100);
+        bLatPnt_00100.setNextButton(bLatPnt_00010);
+        bLatPnt_00010.setNextButton(bLatPnt_00001);
+        bLatPnt_00001.setNextButton(bLonDeg_100);
+
+        bLonDeg_100.setNextButton(bLonDeg_010);
+        bLonDeg_010.setNextButton(bLonDeg_001);
+        bLonDeg_001.setNextButton(bLonMin_10);
+        bLonMin_10.setNextButton(bLonMin_01);
+        bLonMin_01.setNextButton(bLonSec_10);
+        bLonSec_10.setNextButton(bLonSec_01);
+        bLonSec_01.setNextButton(bLonPnt_10000);
+        bLonPnt_10000.setNextButton(bLonPnt_01000);
+        bLonPnt_01000.setNextButton(bLonPnt_00100);
+        bLonPnt_00100.setNextButton(bLonPnt_00010);
+        bLonPnt_00010.setNextButton(bLonPnt_00001);
+
+        final Button buttonDone = ButterKnife.findById(v, R.id.done);
+        if (noTitle) {
+            buttonDone.setVisibility(View.GONE);
+        } else {
+            buttonDone.setOnClickListener(inputdone);
+        }
+
+        loadCalcState();
+        resortEquations();
+        updateResult();
+
+        /**

```

Comment:
Comment not necessary (or if you think that it is necessary, just use a small `//`).